docs/manual/common/gettingstarted/IDEs.md

```diff
@@ -84,9 +88,9 @@ Now update all projects declarations in your Lagom build to use the newly create
 
 Finally, you will need to regenerate the Eclipse project files to account for the changes made in the build file. Go back to the sbt console, type `reload` to reload the build file, and type `eclipse`. Once the `eclipse` task completes, go back to Eclipse, hit F5 to refresh all previously imported projects, and you are done.
 
-### IntelliJ
+### <a name="sbt-intellij"></a>Importing sbt projects into IntelliJ
```

Comment:
I think this should be promoted to a level-2 header like "Importing sbt projects into Eclipse" was.